SKILL.tsv

```diff
@@ -5734,302 +5734,302 @@ SKILL_20151102_005733	$[Merkabah] petrifies [Cursed] enemies for 3 seconds.
+SKILL_20151224_005931	Porapore
+SKILL_20151224_005932	When stacked 100 times, increase holy and dark element by 6% of magic defense for 10 seconds.
+SKILL_20151224_005933	Porapore Activation
+SKILL_20151224_005934	Increase light and dark elemental attack
+SKILL_20151224_005935	Increase magical defense and decrease physical defense
+SKILL_20151224_005936	Spell Breaker
+SKILL_20151224_005937	HP won't be reduced by attacks until all the protections are used
+SKILL_20151224_005938	Shock Cutter
+SKILL_20151224_005939	Apply 5 hits to basic attack for every 5 [Bleeding].
+SKILL_20151224_005940	Shock Cutter: 5 Continuous Attack
+SKILL_20151224_005941	Changed to 5 continuous attacks
+SKILL_20151224_005942	Stamina reduction
+SKILL_20151224_005943	Cripple
+SKILL_20151224_005944	Token benefit: increase moving speed
+SKILL_20151224_005945	Moving speed increased
+SKILL_20151224_005946	Token
+SKILL_20151224_005947	 - Train 2 attribute skills at same time{nl} - Reduce market tax(30%{img white_right_arrow 16 16}10%){nl} - List maximum of 5 items on the market{nl} - Character moving speed +3{nl} - 1:1 trade enabled between token users{nl} - Purchasing shop enabled
+SKILL_20151224_005948	Nexon PC Bang
+SKILL_20151224_005949	Additional +30% Experience gained when defeating enemies{nl}Mark tax reduced by 20%{nl}Item listing count on market increased{nl}Train 3 attribute skills at same time{nl}Character Moving speed +3{nl}Stamina recovery doubled
+SKILL_20151224_005950	Book of Experience
+SKILL_20151224_005951	Additional +30% Experience gained when defeating enemies{nl}Stamina recovery doubled
+SKILL_20151224_005952	Enables you to equip the [Pistol] weapon type. You can also press the C button to use attack.
+SKILL_20151224_005953	[Enables you to equip [Cloth] armor. When equipped with 3 or more pieces of [Cloth] armor, increases your maximum SP and magic damage taken is decreased by 5%. When equipped with 4 [Cloth] armor pieces, your Magic Defense increases and magic damage taken is decreased by an additional 10%.
+SKILL_20151224_005954	[Enables you to equip [Leather] armor. When equipped with 3 or more pieces of [Leather] armor, increases your Evasion. When equipped with 4 [Leather] armor pieces, your Evasion increases from attributes by an additional 50%.
+SKILL_20151224_005955	Enables you to equip [Plate] armor. When equipped with 3 or more pieces of [Plate] armor, increases your maximum HP and physical damage taken is decreased by 5%. When equipped with 4 [Plate] armor pieces, your maximum Stamina increases and physical damage taken is decreased by an additional 10%.
+SKILL_20151224_005956	Stamina consumption cycle duration of companion is increased by 10 seconds per attribute level.
+SKILL_20151224_005957	If companion HP is less than 50%, activate [응급치료] to recover HP for 2 seconds. The duration of the recovery is increased by 1 second per attribute level. [응급치료] can't be used for another 10 minutes after activation
```

Comment:
When the HP of your companion is less than 50%, it will activate the [First Aid] effect and recover HP for 2 seconds. The duration is increased by 1 second per attribute level from Lv. 2.[First Aid] cannot be used for another 10 minutes after it has been activated.